google/cloud/aiplatform/training_jobs.py

```diff
@@ -1153,7 +1176,11 @@ def run(
             training_fraction_split=training_fraction_split,
             validation_fraction_split=validation_fraction_split,
             test_fraction_split=test_fraction_split,
+            predefined_split_column_name=predefined_split_column_name,
             model=managed_model,
+            gcs_destination=gca_io.GcsDestination(
```

Comment:
Prefer to pass the gcs string and create the GcsDestination proto in the base class.

Reply:
Will do

Reply:
I called it `gcs_destination_uri_prefix`